index.js

```diff
@@ -182,6 +182,8 @@ EasyXml.prototype.parseChildElement = function(parentXmlNode, parentObjectNode)
                     } else {
                         parentXmlNode.set(key.substring(1), child);
                     }
+                } else if (typeof child === 'object' && typeof child.toString === 'function') {
```

Comment:
Can you submit this as a separate issue?

I had to remove that line from your PR as it broke a few tests. I'll try to tackle it again.